src/main/java/com/rtb/projectmanagementtool/project/ProjectData.java

```diff
@@ -0,0 +1,204 @@
+/**
+ * ProjectData.java - this file implements the Project class, which contains details about a
+ * particular project, including the members and tasks within it.
+ */
+package com.rtb.projectmanagementtool.project;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import java.lang.reflect.Type;
+import java.util.HashMap;
+import java.util.HashSet;
+
+enum User {
+  CREATOR,
+  ADMIN,
```

Comment:
what's the difference between a creator and admin? might be useful to add a few comments here explaining each type of User.